data-extraction/pyDataExtraction/commonTypes/Graph.py

```diff
@@ -0,0 +1,76 @@
+from typing import Union, Dict, Optional
+from pyDataExtraction.commonTypes.base import DataType
+
+# from docs
+"""
+interface NodeGraphData {
+	id: string;
+	label?: string;
+	color?: string;
+	shape?: "ellipse" | "box";
+}
+"""
+"""
+interface EdgeGraphData {
+	from: string;
+	to: string;
+	label?: string;
+	id?: string;
+	color?: string;
+	dashes?: boolean;
+}
+"""
+# NOTE: may not be able to encapsulate edge in separate class due to from being a syntax token
+"""
+class Edge:
+    
+    def __init__(self,from: str, to: str,):
+        self.fromnode
+        
+    def __repr__(self):
+        pass
```

Comment:
Why have an empty method?

Reply:
because it needs to be implemented. This was mainly meant to be a skeleton of a library, make it easier for people to contribute to the development of one set of dataTypes. Also we can't use the dataType's `__repr__` method for Edge or Node. 

for one, json complains Node  objects aren't serializable when calling `__repr__` on graph. 

two, `from` is a syntax token, and also what is expected to be inside the json representation of a graph. so in Edge's case in particular, it's going to require us to either do something hacky to use from as a keyword, or parse the json representation of it's dict and change every instance of whatever variable we used to represent `from`